File: src/sdk-reference/cpp/1/collection/get-specifications/snippets/get-specifications.cpp
try {
  std::string specifications = kuzzle->collection->getSpecifications("nyc-open-data", "yellow-taxi");

  std::cout << specifications << std::endl;
  // {"nyc-open-data": {"yellow-taxi": {"strict": false, "fields": {"license": {"type": "string"}}}}}
} catch (kuzzleio::KuzzleException &e) {
  std::cerr << e.what() << std::endl;
}

File: src/sdk-reference/csharp/1/collection/get-specifications/snippets/get-specifications.cs
try {
  string specifications = kuzzle.collection.getSpecifications("nyc-open-data", "yellow-taxi");

  Console.WriteLine(specifications);
  // {"nyc-open-data": {"yellow-taxi": {"strict": false, "fields": {"license": {"type": "string"}}}}}
} catch (KuzzleException e) {
  Console.Error.WriteLine(e.what());
}
